lib/runtime-core/src/fault.rs

```diff
@@ -40,6 +40,7 @@ struct UnwindInfo {
 
 thread_local! {
     static UNWIND: UnsafeCell<Option<UnwindInfo>> = UnsafeCell::new(None);
+    static CURRENT_CTX: UnsafeCell<*mut vm::Ctx> = UnsafeCell::new(::std::ptr::null_mut());
```

Comment:
I don't presently understand what it means for there to ever be more than one context?

Reply:
In case of nested WASM invocations:

Native -> WASM -> (import function) Native -> WASM